tests/migrations/test_loader.py

```diff
@@ -521,7 +521,12 @@ def test_invalid(self):
         MigrationLoader reraises ImportErrors caused by "bad magic number" pyc
         files with a more helpful message.
         """
-        with self.temporary_migration_module(module='migrations.test_migrations_bad_pyc'):
+        with self.temporary_migration_module(module='migrations.test_migrations_bad_pyc') as migration_dir:
+            # The -tpl suffix is to avoid the pyc exclusion in MANIFEST.in.
+            os.rename(
```

Comment:
Hm, shouldn't this be a *copy* operation? Otherwise it will fail the second time..?

Reply:
`temporary_migration_module` makes a copy each time.